save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt

```diff
@@ -153,18 +152,13 @@ internal class OrganizationAdminView : AbstractView<Props, OrganizationAdminStat
      */
     private suspend fun getOrganizations(): MutableList<Organization> {
         val response = get(
-            url = "$apiUrl/organizations/all",
+            url = "$apiUrl/organizations/all?includeDeleted=${false}",
```

Comment:
why `${false}`? do we need to calculate it?

Reply:
Mainly to avoid mistyping when copying or refactoring.